multicluster/controllers/multicluster/stale_controller.go

```diff
@@ -43,17 +43,20 @@ type StaleController struct {
 	client.Client
```

Comment:
Update the comments before StaleController (maybe name it to StaleResCleanupController? We can keep the file name)